modules/scripts/startup-script/variables.tf

```diff
@@ -112,6 +112,17 @@ variable "install_ansible" {
   default     = null
 }
 
+variable "configure_ssh" {
```

Comment:
Batch calls this `permissiveSsh`. I would suggest `configure_permissive_ssh` or `enable_permissive_ssh` for this var name.

Reply:
It is not quite permissive, I think we are not introducing any security issue here. So it is really more about configuring ssh.